src/components/m3Dialog/m3Dialog.styles.scss

```
@@ -0,0 +1,263 @@
+		padding: 0;
+		border: 0;
+		border-radius: 50%;
+		background: transparent;
+		color: var(--m3-on-surface-variant, #444748);
+		cursor: pointer;
+		transition: background-color 150ms ease;
+
+		svg {
+			width: 16px;
+			height: 16px;
+			fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)